### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by kaushikb11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,112 @@
...
+
+</div>
+</div>
+
+______________________________________________________________________
+
+
+# API Access Frontend Component
+
+This component allows you to enable users to engage with your app via an API. It is a simple component that allows you to create an UI guide on how to send requests to your App APIs. The component is built with [Lightning's StaticWebFrontend](https://lightning.ai/lightning-docs/api_reference/generated/lightning_app.frontend.web.StaticWebFrontend.html) and [React](https://reactjs.org/) for the interface.
```
Comment (ethanwharris) [inline README.md]: Would maybe simplify:
```suggestion
This component allows you to create a UI guide on how to send requests to your App APIs. The component is built with [Lightning's StaticWebFrontend](https://lightning.ai/lightning-docs/api_reference/generated/lightning_app.frontend.web.StaticWebFrontend.html) and [React](https://reactjs.org/) for the interface.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,112 @@
...
+
+
+## Getting Started
+
+### Installation
+
+```bash
+lightning install component git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git@main
```
Comment (ethanwharris) [inline README.md:39]: Cool, does this actually work? Let's also remember to update it once the component is in the gallery

File: README.md [hunk 3]
```diff
@@ -1 +1,112 @@
...
+```bash
+git clone https://github.com/Lightning-AI/LAI-API-Access-UI-Component
+
+cd LAI-API-Access-UI-Component
+pip install .
+```
+
+### Usage
+
+Here's a small [snippet](https://github.com/Lightning-AI/LAI-API-Access-UI-Component/blob/main/examples/example_app.py#L9) of how to use the component in your App.
```
Comment (ethanwharris) [inline README.md]:
```suggestion
Here's a small [snippet](https://github.com/Lightning-AI/LAI-API-Access-UI-Component/blob/main/examples/example_app.py#L9) showing how to use the component in your App:
```

**Review by ethanwharris — APPROVED** [review]
LGTM, small suggestions :smiley:

File: README.md (hunk 2 above)
Comment (kaushikb11) [reply README.md:39]:
```
❯ lightning install component git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git@main >> intall_logs.txt
INFO: 
    ⚡ Installing Lightning component ⚡

    component name : LAI-API-Access-UI-Component
    developer      : Lightning-AI

    ⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡
    WARNING: this is NOT an official Lightning Gallery component
    Install at your own risk
    ⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡⚡

    Installation runs the following command for you:

    pip install git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git@main
    

INFO: ⚡ RUN: pip install
  Running command git clone --filter=blob:none --quiet https://github.com/Lightning-AI/LAI-API-Access-UI-Component.git /private/var/folders/3j/h7yk_n256r15z7ttg578k8lw0000gn/T/pip-req-build-o6fl4y5f

[notice] A new release of pip available: 22.2.2 -> 22.3
[notice] To update, run: pip install --upgrade pip
INFO: 
        ⚡ Installed! ⚡

        to use your component:
        from the_component import TheClass

        make sure to add this entry to your Lightning APP requirements.txt file:
        {git_url}

        if you want to uninstall, run this command:
        pip uninstall {uninstall_step}
```
yup, here are the CLI logs

**Pull request merged**